### PR TITLE
Display value map

### PR DESCRIPTION
**Pull request by dariuszsobkowicz**
- display valueMap
- fixed filters border rendering
- update conseiljs

### CODE REVIEW

File: .prettierrc [hunk 1]
```diff
@@ -1,5 +1,7 @@
 {
   "singleQuote": true,
+  "printWidth": 160,
+  "tabWidth": 2,
```
Comment (anonymoussprocket) [inline .prettierrc]: should be 4

File: .prettierrc [hunk 2]
```diff
@@ -1,5 +1,7 @@
 {
```
Comment (anonymoussprocket) [inline .prettierrc]: Maybe consolidate these into the eslint rules in package.json?

File: src/utils/render.tsx [hunk 3]
```diff
@@ -35,131 +35,156 @@ const LinkSpan = styled.span`
...
-                {address}
-                <Clipboard value={value} />
-            </React.Fragment>
-        );
-    } else if (dataType === AttrbuteDataType.HASH) {
-        const hash = formatReferenceValue(attribute, (truncate ? truncateHash(value) : value), value, onClickPrimaryKey);
+    return errors.join(', ');
+  }
+
+  if (!!aggregation) return formatAggregatedValue(attribute, value, aggregation);
```
Comment (anonymoussprocket) [inline src/utils/render.tsx]: Always curly braces!

File: .prettierrc (hunk 2 above)
Comment (dariuszsobkowicz) [reply .prettierrc]: that's need some extra configuration to work together, I suggest to create task

File: src/utils/render.tsx [hunk 4]
```diff
@@ -12,32 +12,32 @@ import { AttributeDefinition, AttrbuteDataType, ConseilFunction } from 'conseilj
...
+    color: #56c2d9;
+    cursor: pointer;
+    text-decoration: underline;
 `;
 
 const PrimaryKeyList: any = {
     blocks: ['hash', 'level'],
     accounts: ['account_id'],
-    operations: ['operation_group_hash']
+    operations: ['operation_group_hash'],
```
Comment (anonymoussprocket) [inline src/utils/render.tsx]: avoid trailing commas

**Pull request merged**